setup.py

```diff
@@ -15,15 +15,17 @@
     platforms='any',
     install_requires=[
         'tornado',
-        'opentracing>=2.0,<2.1',
+        'opentracing>=2.0,<=2.3',
```

Comment:
Why not accept 2.3 maintenance releases?

Reply:
Mistake. That was the intention. Will fix.